src/Options/Definitions.js

```diff
@@ -360,6 +360,11 @@ module.exports.ParseServerOptions = {
     action: parsers.numberParser('schemaCacheTTL'),
     default: 5000,
   },
+  securityChecks: {
+    env: 'PARSE_SERVER_SECURITY_CHECKS_OPTIONS',
```

Comment:
DX: What about just `audit` since this feature audit the parse server and the log the report :)  I think the `audit` key word is well appropriate here in a security context like `npm audit`

Reply:
There has already been a [discussion](https://github.com/parse-community/parse-server/pull/6973#issuecomment-723593252) about terminology:
- Terms like `advisory` or `audit` are too general and can create confusion / collide with other feature descriptions.
- Using terms of similar features in related 3rd party services can be a disadvantage in discussion and online search as the terms are too similar. Distinct terms that clearly communicate the purpose create an advantage in this regard.

Reply:
Please see [this comment](https://github.com/parse-community/parse-server/pull/6973#issuecomment-745483433).